spec/linter-htmlhint-spec.js

```diff
@@ -32,4 +32,26 @@ describe('The htmlhint provider for Linter', () => {
 
     expect(messages.length).toBe(0);
   });
+
+  it('lints an invalid file (bad.html) with a default configuration when no config file is present', async () => {
```

Comment:
I'd put both of these in a `describe` block myself.

Reply:
How would you envision that? With a nested `describe` or a new one, and what would its description be?

Reply:
Nested under the one from L9 of course, and something like:

```js
describe("The 'Disable when no HTMLHint config is found' option", () => {
  it('lints files with no config when disabled', async () => {
    // ..
  });

  it("doesn't lint files with no config when enabled", async () => {
    // ...
  });
});
```